coderd/parameter/plaintext_test.go

```diff
@@ -0,0 +1,49 @@
+package parameter_test
+
+import (
+	"testing"
+
+	"github.com/coder/coder/coderd/parameter"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPlaintext(t *testing.T) {
+	t.Parallel()
+	t.Run("Simple", func(t *testing.T) {
+		t.Parallel()
+
+		mdDescription := `# Provide the machine image
+See the [registry](https://container.registry.blah/namespace) for options.
+
+![Minion](https://octodex.github.com/images/minion.png)
+
+**This is bold text.**
+__This is bold text.__
+*This is italic text.*
+> Blockquotes can also be nested.
+~~Strikethrough.~~
+
+1. Lorem ipsum dolor sit amet.
+2. Consectetur adipiscing elit.
+3. Integer molestie lorem at massa.
+
+` + "`There are also code tags!`"
+
+		expected := "Provide the machine image See the registry for options. Minion This is bold text. This is bold text. This is italic text. Blockquotes can also be nested. Strikethrough. 1. Lorem ipsum dolor sit amet. 2. Consectetur adipiscing elit. 3. Integer molestie lorem at massa. There are also code tags!"
```

Comment:
Do we actually want the CLI output to be this lossy? For instance:
- Why do we not keep newlines?
- Why don't we print links? We could e.g. do something like: `See the registry (https://container.registry.blah/namespace) for options`.

If newlines are not wanted here, perhaps we could call the field `DescriptionCondensed` instead of plaintext?

Reply:
Good call, mate! We can safely leave them.

<img width="531" alt="Screenshot 2023-02-08 at 12 39 59" src="https://user-images.githubusercontent.com/14044910/217520382-1950fcf4-90b1-42b3-98a7-18594228601d.png">